### DISCUSSION TITLE
Updating map by external triggers

### DISCUSSION THREAD

**Discussion by dimitryvolos** [themustafaomar/jsvectormap#172] [Q&A]
Hi,

Is it possible to zoom in and out of specific regions/markers by clicking the external button?

For example, I can use 'map.setBackgroundColor("#")' to modify the map. Can I use 'map.focusOn' to zoom in on a specific region by clicking on element that is is not part of the map.

I have a simple table listing regions. When I click on GB, I would like to zoom in on 'GB' and the appropriate marker selected.

I would appreciate any help.

**Comment by themustafaomar** [comment]
Hi @dimitryvolos very sorry for the late response.

Yes you can do that, here is an example you may want to take a look: https://codepen.io/themustafaomar/pen/rNXbxQR
